src/antares/service/api_services/output_api.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2024, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+
+
+from antares.api_conf.api_conf import APIconf
+from antares.api_conf.request_wrapper import RequestWrapper
+from antares.exceptions.exceptions import APIError, OutputsRetrievalError
+from antares.model.output import Output
+from antares.service.base_services import BaseOutputService
+
+
+class OutputApiService(BaseOutputService):
+    def __init__(self, config: APIconf, study_id: str):
+        super().__init__()
+        self.config = config
+        self.study_id = study_id
+        self._base_url = f"{self.config.get_host()}/api/v1"
+        self._wrapper = RequestWrapper(self.config.set_up_api_conf())
+
+    def read_outputs(self) -> list[Output]:
+        url = f"{self._base_url}/studies/{self.study_id}/outputs"
+        try:
+            response = self._wrapper.get(url)
+            outputs_json_list = response.json()
+            if isinstance(outputs_json_list, dict):
```

Comment:
the endpoint is supposed to return a list so I don't see where you could enter this if

Reply:
My bad i misread the api doc